### DISCUSSION TITLE
卡饭 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#10] [模板讨论]
卡饭 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by jerrell851** [comment]
卡饭加了浏览器检测，请修改一下，谢谢。

**Reply by wjf0214** [reply]
网站刚改的时候我看了代码。过盾不麻烦。但是用框架实现起来麻烦。因为要遍历可能几十万次。操作起来不太方便。框架也有速率限制。

**Reply by wjf0214** [reply]
卡饭，我刚测试，盾已经去掉了。
